db/migrate/20210709145249_create_hearing_email_recipient.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class CreateHearingEmailRecipient < Caseflow::Migration
+  def change
+    create_table :hearing_email_recipients,
+                 comment: "Add reminder email recipients for non-virtual hearings" do |t|
```

Comment:
```suggestion
                 comment: "Recipients of hearings-related emails" do |t|
```
Describes what it is rather than what we were doing when we created it